coopr-docs/docs/source/rest/metrics.rst

```diff
@@ -81,3 +83,180 @@ Example
+
+.. code-block:: bash
+
+ {"start":1424475097,"end":1424486443,"data":[{"time":1424475097,"value":31872}]}
+ {"start":1424475097,"end":1424486503,"data":[{"time":1424475097,"value":32112}]}
+
+
+
+Time Ranges
+-----------
+
+Start and end values are expressed in Unix time (a.k.a. seconds since epoch, or the beginning of Unix time:
+00:00:00 Coordinated Universal Time (UTC), Thursday, 1 January 1970).
+Unix time may be checked on most Unix systems by typing ``date +%s`` on the command line.
+
+With the general usage example, the JSON response to general node hour usage shows the start and end values returned. 
+If the start is not specified, the start time is the exact date and time when the very first 
+task (ever) completes. The end date, if not specified, is the current date and time.
+
+
+Start Time Only
+^^^^^^^^^^^^^^^
+
+To obtain node usage hours since a specific date and time, we would add ``start=x`` where x is a Unix time.
+
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:admin'
+        -H 'Coopr-ApiKey:<apikey>'
+        -H 'Coopr-TenantID:superadmin'
+        http://<server>:<port>/<version>/metrics/nodes/usage?start=1424393580
+ 
+ $ {"start":1424393580,"end":1424485774,"data":[{"time":1424393580,"value":29199}]}
+
+
+End Time Only
+^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:admin'
+        -H 'Coopr-ApiKey:<apikey>'
+        -H 'Coopr-TenantID:superadmin'
+        http://<server>:<port>/<version>/metrics/nodes/usage?end=1424482780
+ 
+ $ {"start":1424475097,"end":1424482780,"data":[{"time":1424475097,"value":17220}]}
+
+
+Time Range (Start and End Time)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:admin'
+        -H 'Coopr-ApiKey:<apikey>'
+        -H 'Coopr-TenantID:superadmin'
+        http://<server>:<port>/<version>/metrics/nodes/usage?start=1424481097&end=1424482780
+ 
+ $ {"start":1424481097,"end":1424486196,"data":[{"time":1424481097,"value":20396}]}
+
+
+
+Specifying a User
+-----------------
+
+Non-superadmin tenants can retrieve node usage metrics for their users only. The superadmin tenant retrieves
+node usage metrics for all users. As an example, if there are users named ``george`` under both the superadmin 
+and a non-superadmin tenant named ``mytenant``, the ``mytenant`` admin user will only see node usage for their 
+own ``george`` user, while the superadmin will retrieve combined total usage for all users named ``george`` 
+(under all tenants, including superadmin). 
+
+The two scenarios in this example are illustrated here. Node usage metrics for:
+
+1. A user named ``george`` while we authenticate as ``user=admin``, and the ``mytenant`` tenant.
+
+   .. code-block:: bash
+   
+      $ curl -H 'Coopr-UserID:admin'
+             -H 'Coopr-ApiKey:<apikey>'
+             -H 'Coopr-TenantID:mytenant'
+             http://<server>:<port>/<version>/metrics/nodes/usage?user=george
+      
```

Comment:
I have it that way for readability (to make the result easy to distinguish from the api call).